src/meta/src/hummock/manager/mod.rs

```diff
@@ -145,6 +146,14 @@ pub struct HummockManager {
     // `compaction_state` will record the types of compact tasks that can be triggered in `hummock`
     // and suggest types with a certain priority.
     pub compaction_state: CompactionState,
+
+    // Record the partition corresponding to the table in each group (accepting delays)
+    // Currently, we update it in a couple of scenarios
+    // 1. throughput and size are checked periodically and calculated according to the rules
+    // 2. A new group is created (split)
+    // 3. split_weight_by_vnode is modified for an existing group. (not supported yet)
+    group_to_table_vnode_partition:
```

Comment:
Can you also document what does it mean if:
- a table id is not found in this mapping 
- a table id is found in this mapping with 0 as the vnode partition number
- a table id is found in this mapping with 1 as the vnode partition numner